h2o-r/h2o-package/R/communication.R

```diff
@@ -851,7 +851,11 @@ h2o.show_progress <- function() assign("PROGRESS_BAR", TRUE, .pkg.env)
         if(verboseModelScoringHistory){
           cat(paste0("\nScoring History for Model ",job$dest$name, " at ", Sys.time(),"\n"))
           print(paste0("Model Build is ", job$progress*100, "% done..."))
-          print(tail(h2o.getModel(job$dest$name)@model$scoring_history))
+          tryCatch({
```

Comment:
It would be probably a little cleaner to have a function like h2o.exists and check if the model exists first. However this would require another http call in the progress bar. I don't think it is necessary.